src/main/java/rcms/utilities/daqexpert/reasoning/logic/failures/HltOutputBandwidthExtreme.java

```diff
@@ -0,0 +1,63 @@
+package rcms.utilities.daqexpert.reasoning.logic.failures;
+
+import rcms.utilities.daqaggregator.data.DAQ;
+import rcms.utilities.daqexpert.ExpertException;
+import rcms.utilities.daqexpert.ExpertExceptionCode;
+import rcms.utilities.daqexpert.Setting;
+import rcms.utilities.daqexpert.reasoning.base.action.SimpleAction;
+import rcms.utilities.daqexpert.reasoning.logic.basic.Parameterizable;
+
+import java.util.Map;
+import java.util.Properties;
+
+public class HltOutputBandwidthExtreme extends KnownFailure implements Parameterizable {
+
+    /**
+     * upper end of range for expected  rate
+     */
+    private double max;
+
+    public HltOutputBandwidthExtreme() {
+        this.name = "Extreme HLT output bandwidth";
+        this.max = 0;
+
+        this.action = new SimpleAction("Are we running with the correct pre-scale column?",
```

Comment:
in fact, the bandwidth is the product of the (output) trigger rate and the event size. If we add a module checking the HLT output event rate separately in the future, we should move the question about the prescale column there.

In principle a high HLT output rate has also occurred because of a wrong HLT menu put in place (which only the HLT expert can fix) but in practice I think choosing the wrong prescale column happens much more often. So I would leave it as is for the moment.

Another